app/templates/src/main/java/package/social/_SocialService.java

```diff
@@ -0,0 +1,109 @@
+package <%=packageName%>.social;
+
+import <%=packageName%>.domain.Authority;
+import <%=packageName%>.domain.User;
+import <%=packageName%>.repository.AuthorityRepository;
+import <%=packageName%>.repository.UserRepository;
+import <%=packageName%>.service.MailService;
```

Comment:
We usually (not always perfect) have a space between our package name and the other packages